tools/IceRpc.Protobuf.Tools/BuildTelemetryTask.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) ZeroC, Inc.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace IceRpc.Protobuf.Tools;
+
+/// <summary>
+/// A MSBuild task that reports Protobuf build telemetry data to the IceRPC build telemetry service.
+/// </summary>
+public class BuildTelemetryTask : ToolTask
+{
+    /// <summary>
+    /// Gets or sets the compilation hash.
+    /// </summary>
+    [Required]
+    public string CompilationHash { get; set; } = "";
+
+    /// <summary>
+    /// Gets or sets the number of source files in the Slice compilation.
+    /// </summary>
+    public int SourceFileCount { get; set; }
```

Comment:
This should say `protobuf` instead of `Slice` compilation.
Also, I'm not sure whether making a distinction of "source" files is meaningful/correct with Protobuf,
unlike in Slice.